service/src/main/webapp/UniversityPage.jsp

```diff
@@ -0,0 +1,24 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8"
+    pageEncoding="UTF-8" import="java.util.*"%>
+<html>
+    <head>
+        <title>${longName}</title>
+        <link href="${pageContext.request.contextPath}/assets/css/base.css" rel="stylesheet" type="text/css">
+    </head>
+        <body class="four-column">
+        <jsp:include page="header.jsp"/>
+        <div class="column-beta">
+                <h1><%= request.getAttribute("longName")%></h1>
```

Comment:
These should not be passed this way. University information should be gotten by looking up a university by it's ID via the objectmodel code.

ID should be given via URL: *www.fweasel.com/university?uid=435*

If objectmodel code requires code changes let us know and we can fix it.